code/modules/projectiles/guns/energy/energy_gun.dm

```diff
@@ -53,9 +53,10 @@
 /obj/item/gun/energy/e_gun/hos
 	name = "\improper X-01 MultiPhase Energy Gun"
 	desc = "This is an expensive, modern recreation of an antique laser gun. This gun has several unique firemodes, but lacks the ability to recharge over time."
+	cell_type = /obj/item/stock_parts/cell{charge = 1200; maxcharge = 1200}
```

Comment:
Gross.

Max an actual subtype.

Reply:
There's no way to remove it, and it's not actually used in crafting it. Mini eguns already do this.

Reply:
Existing bad code is not justification for future bad code.

A number of things aren't interactable by players, but none the less, explicit subtypes are defined. Plus. Who knows--maybe this will be used somewhere else, one day.

Reply:
This isn't worth bikeshedding over in a PR with enormous balance implications.

Reply:
and for the record, stop using the type path system as a value database, it's awful